main.go

```diff
@@ -100,11 +102,12 @@ func snapshot() error {
 		}
 
 		labels := prometheus.Labels{
-			"number":   strconv.Itoa(prInfo.Number),
-			"label":    strings.Join(labelsTag, ","),
-			"author":   prInfo.User,
-			"reviewer": strings.Join(reviewersTag, ","),
-			"repo":     prInfo.Repo,
+			"number":        strconv.Itoa(prInfo.Number),
+			"label":         strings.Join(labelsTag, ","),
+			"author":        prInfo.User,
+			"reviewer":      strings.Join(reviewersTag, ","),
+			"repo":          prInfo.Repo,
+			"lifetime_days": strconv.Itoa(int(now.Sub(prInfo.CreatedAt).Hours() / 24)),
```

Comment:
Is it difficult to calculate the average or something if I just add it to the label like this?
I am not very familiar with datadog and would appreciate your advice.

Reply:
Hi, @YutaUra Thank you for your contribution!

In general, from a cost perspective, it is better not to have too many different types of tags. 
In this case, the number of tag types will increase with each passing day.

In your use case, I assume that you want to determine which "n" days have passed.
Then, why not use a value: `lifetime_status` and give `stale` to those that are "n" days old and `ok` to those that are not as value?

Also, I would like to be able to get the "n" days from an environment variable such as LIFETIME_STALE_DAYS, and default to 14 days if appropriate.

Could you please make these changes? Thanks.

Reply:
Thank you as well!
And thanks for the advice! I'm so glad I didn't think about it from a cost perspective.
I've made some changes and I'd like you to check them out!

6dd91632a0615ee2fe31cac7552db77315812ab5